web/themes/custom/pineapple/components/_field.twig

```diff
@@ -37,6 +37,7 @@
  * @see template_preprocess_field()
  */
 #}
+{{ attach_library('pineapple/clickable-card') }}
```

Comment:
Why are you attaching this library here?

Reply:
my previous code required it but it's removed now